app/controllers/trainees/diversity/ethnic_backgrounds_controller.rb

```diff
@@ -26,13 +27,21 @@ def trainee
         @trainee ||= Trainee.find(params[:trainee_id])
       end
 
-      def ethnic_background_param
+      def ethnic_background_params
         return { ethnic_background: nil } if params[:diversities_ethnic_background_form].blank?
 
         params.require(:diversities_ethnic_background_form).permit(
           *Diversities::EthnicBackgroundForm::FIELDS,
         )
       end
+
+      def reset_attributes(params)
+        if params[:ethnic_background] != trainee.ethnic_background
```

Comment:
What if I selected the wrong `ethnic_background` to begin with? If I want to change from "Asian or Asian British" to "Mixed or multiple ethnic groups" for example, I then cannot save a new `additional_ethnic_background`.

Reply:
Fixed in https://github.com/DFE-Digital/register-trainee-teachers/pull/313/commits/7dd83e8eb8dbc1b32b6867166213fba676cbccf5